package.json

```
@@ -34,45 +32,35 @@
     "stat-mode": "0.3.0",
     "stream-to-promise": "2.2.0",
     "tar": "4.4.18",
+    "tinyexec": "0.3.2",
     "tree-kill": "1.2.2",
     "uid-promise": "1.0.0",
     "uuid": "3.3.2",
     "xdg-app-paths": "5.1.0",
     "yauzl-promise": "2.1.3"
   },
   "devDependencies": {
+    "@esbuild/darwin-arm64": "^0.24.2",
     "@types/debug": "4.1.9",
-    "@types/fs-extra": "8.1.0",
     "@types/generic-pool": "3.1.9",
-    "@types/jest": "27.0.2",
     "@types/micro": "7.3.6",
     "@types/node": "16.18.122",
     "@types/node-fetch": "2.5.0",
     "@types/tar": "4.0.0",
     "@types/uuid": "8.3.1",
     "@types/yauzl-promise": "2.1.0",
+    "@ungap/structured-clone": "^1.3.0",
```

```suggestion
    "@ungap/structured-clone": "1.3.0",
```
Pin dependency to specific version.